packages/itwin/tree-widget/src/components/trees/models-tree/ModelsTreeButtons.tsx

```diff
@@ -180,19 +180,21 @@ export function View2DButton(props: ModelsTreeHeaderButtonProps) {
   }, [models2d, props.viewport]);
 
   return (
-    <Button
-      size={props.density === "enlarged" ? "large" : "small"}
-      styleType="borderless"
-      title={TreeWidget.translate("modelsTree.buttons.toggle2d.tooltip")}
-      onClick={() => {
-        props.onFeatureUsed?.("models-tree-view2d");
-        void toggleModels(models2d, is2dToggleActive, props.viewport);
-      }}
-      disabled={models2d.length === 0}
-      endIcon={is2dToggleActive ? <SvgVisibilityShow /> : <SvgVisibilityHide />}
-    >
-      {TreeWidget.translate("modelsTree.buttons.toggle2d.label")}
-    </Button>
+    <Tooltip content={TreeWidget.translate("modelsTree.buttons.toggle2d.tooltip")}>
+      <Button
+        size={props.density === "enlarged" ? "large" : "small"}
+        styleType="borderless"
+        title={TreeWidget.translate("modelsTree.buttons.toggle2d.tooltip")}
```

Comment:
Same as with checkbox `title` should not be used.

Reply:
removed